charts/fybrik/templates/fybrik-blueprints-np.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if include "fybrik.isEnabled" (tuple .Values.manager.enabled .Values.worker.enabled) }}
+{{- if .Values.global.isNPEnabled }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
```

Comment:
Not related to this file but I am missing RBAC permissions for the manager to be able to apply/modify/delete NetworkPolicy resources.

Reply:
You don't, manager has permissions to do everything

Reply:
This file, sets default policy that no one can connect to the pods (modules) in the namespace, except explicit added permissions. The networkkpolicies that are created dynamically.